packages/core/src/render3/interfaces/definition.ts

```diff
@@ -211,8 +211,11 @@ export interface DirectiveDef<T> {
    * Patched onto the definition by the `HostDirectivesFeature`.
    */
   applyHostDirectives:
-      ((tView: TView, viewData: LView, tNode: TElementNode|TContainerNode|TElementContainerNode,
-        matches: any[]) => void)|null;
+      ((matches: DirectiveDef<unknown>[], def: DirectiveDef<unknown>, tView: TView, viewData: LView,
```

Comment:
I think it's more common in the code to name `LView` instances as `lView`:
```suggestion
      ((matches: DirectiveDef<unknown>[], def: DirectiveDef<unknown>, tView: TView, lView: LView,
```